playdate_example/config.nims

```diff
@@ -1 +1,3 @@
 include playdate/build/config
+
+switch("d", "memtrace")
```

Comment:
Maybe add a comment with explanation

Reply:
I may just remove this. It’s only for internal debugging of the allocator.